packages/tests/tests/store/utils.ts

```diff
@@ -107,16 +107,27 @@ export async function startAndConnectLightNode(
   shardInfo?: ShardingParams
 ): Promise<LightNode> {
   const waku = await createLightNode({
+    pubsubTopics: shardInfo ? undefined : pubsubTopics,
+    staticNoiseKey: NOISE_KEY_1,
+    libp2p: { addresses: { listen: ["/ip4/0.0.0.0/tcp/0/ws"] } },
     ...((pubsubTopics.length !== 1 ||
       pubsubTopics[0] !== DefaultPubsubTopic) && {
       shardInfo: shardInfo
-    }),
-    pubsubTopics: shardInfo ? undefined : pubsubTopics,
-    staticNoiseKey: NOISE_KEY_1
+    })
   });
   await waku.start();
   await waku.dial(await instance.getMultiaddrWithId());
   await waitForRemotePeer(waku, [Protocols.Store]);
+
+  const wakuPeers = await waku.libp2p.peerStore.all();
```

Comment:
are you attempting to find connected nodes or discovered nodes?

Reply:
That's a good question. I think both are enough as in test environment we expect only our nodes to be present, and if they are present - they should be connected.

Reply:
hm since on the preceeding line you're waitingForRemotePeer, you might wanna find connected peers instead

Reply:
moving it to `getConnections`